playground/tei_bot.py

```diff
@@ -42,95 +40,71 @@ def array_to_stone(stone_array: np.array, color: str) -> List[Stone]:
             out = 2
         else:
             out = -1
-
         y, x = np.where(stone_array == out)
         positions = list(zip(list(x), list(y)))
         stones = [Stone(str(x + 1), str(y + 1), color) for x, y in positions]
-
         return stones
 
     @staticmethod
-    def move_stone(stone, direction, stride=1):
+    def move_stone(stone: Stone, direction: str, stride: int = 1) -> Stone:
         x = int(stone.x)
         y = int(stone.y)
 
-        if direction == 'l':
+        if direction == "l":
```

Comment:
directions might be widely used in other places, 
so maybe we can separate as constants, standardize, and use them: https://www.edureka.co/community/16286/create-a-constant-in-python

Reply:
plus, making as a constant might be easier to understand at first glance than just a single letter(e.g.  `l`,`r`)

Reply:
agree.